samples/typescript_nodejs/57.teams-conversation-bot/src/teamsConversationBot.ts

```diff
@@ -158,18 +160,34 @@ export class TeamsConversationBot extends TeamsActivityHandler {
             console.log( 'a ', teamMember );
             const message = MessageFactory.text( `Hello ${ teamMember.givenName } ${ teamMember.surname }. I'm a Teams conversation bot.` );
 
-            const ref = TurnContext.getConversationReference( context.activity );
-            ref.user = teamMember;
-            let botAdapter: BotFrameworkAdapter;
-            botAdapter = context.adapter as BotFrameworkAdapter;
-            await botAdapter.createConversation ( ref,
-                async ( t1 ) => {
-                    const ref2 = TurnContext.getConversationReference( t1.activity );
-                    await t1.adapter.continueConversation( ref2, async ( t2 ) => {
-                        await t2.sendActivity( message );
-                    } );
-                } );
-        } );
+            // const ref = TurnContext.getConversationReference( context.activity );
+            // ref.user = teamMember;
+            let botAdapter: CloudAdapter;
+            botAdapter = context.adapter as CloudAdapter;
+
```

Comment:
Should we remove the commented lines?
For the `botAdapter` variable, do we need this variable to be cast to CloudAdapter, we can't simply do `context.adapter` like in line 183, because of the `BotAdapter` interface

Reply:
You are right. Fixed. Thanks!